### DISCUSSION TITLE
Pin code changed

### DISCUSSION THREAD

**Discussion by Rahim407** [reiryuki/Dolby-Atmos-OnePlus-9-Pro-Magisk-Module#2] [Announcements]
After installing this modul on my Pixel 4 with ElexirOs A13 passcode changed and i can use my phone. Also, i cant delete module neither through twrp(because i dont have it) nor through adb.

**Comment by g7755726** [comment]
Yeah this happened to me but it's temporary you will need to disable the module to regain access to your pin, luckily I had a anti bootloop module to disable the module

**Comment by reiryuki** [comment]
Please rea the Installation Guide Kitsune Mask
